vm/src/lib.rs

```diff
@@ -290,6 +290,10 @@ impl<'a> VirtualMachine<'a> {
             self.monitor.enabled = true;
         }
     }
+    
+    pub fn in_stepping_mode(&self) -> bool {
```

Comment:
Sure, seems a bit more clear.
But I don't really know how to do it from here, since it's already merged? Should I open a new PR for this?